speed-transition.lua

```diff
@@ -63,34 +84,47 @@ end
 
 function skipval()
    local skipval = mp.get_property_native("demuxer-cache-duration", 5)
-   skipval = clamp(skipval, 1, 5)
    if nextsub > 0 then
-      if nextsub-skipval-leadin <= 0 or directskip then
-         skipval = nextsub - leadin
+      if directskip then
+         skipval =  nextsub - leadin
+      elseif nextsub-skipval-leadin <= 0 then
+         skipval =  clamp(nextsub-leadin, 0, maxskip)
```

Comment:
now that maxskip is a configurable option, this logic had to be broken up. Also, skipdelay allows clamp MIN value to be zero.